src/resolvers/programmeResolvers.ts

```diff
@@ -145,11 +145,12 @@ export const resolvers = {
     async subjects(category: GQLCategory, __: any, context: ContextWithLoaders): Promise<GQLSubject[]> {
       const children = await fetchChildren({ id: category.id, nodeType: "SUBJECT" }, context);
       const nodes = children.map((child) => {
-        const context = child.contexts.find((c) => c.path.startsWith("/subject")) || child.contexts[0];
+        const context = child.contexts.find((c) => c.rootId === child.id) || child.contexts[0];
```

Comment:
Kan vi ikke stole på `child.context` her?

Reply:
Nei, for fag som ligger under programområder så ønsker du konteksten som har faget som rot. Siden du henter fag som barn av et program, vil du få en kontekst tilsvarende /programme/programme/programme/subject og det vil vi ikkje.

Reply:
Da er det nok lurt å legge på en kommentar!

Reply:
Hvorfor kan vi ikke gjøre dette i taksonomi?

Reply:
Akkurat denne er spesiell fordi du henter barn av programfagnode. Då velger taksonomi konteksten som har programfaget som rot, og då får du pather på formatet /programme:*/programme:*/.... 
Ved å velge en kontekst som har faget sjølv som rot får du korrekte urler for faget.

Reply:
Kan vi ikke spesialhåndtere den i taksonomi, isåfall? Finnes det tilfeller der vi faktisk vil ha `/programme:/programme:/`?

Reply:
I alle andre tilfeller ønsker du at barna du henter er relatert til noden som sendes inn, og taksonomi veit ikkje kva du ønsker deg, så akkurat denne vil eg la stå.

Reply:
Den er god, tenkte bare det hadde vært greit å spesialhåndtere programfagsnoder!